open_src/influx/meta/data.go

```diff
@@ -1370,6 +1384,45 @@ func (data *Data) ShowRetentionPolicies(database string) (models.Rows, error) {
 	return []*models.Row{row}, nil
 }
 
+func (data *Data) ShowClusterStatus() models.Rows {
+	row := &models.Row{Columns: []string{"time", "status", "hostname", "nodeID", "nodeType"}}
+	timestamp := time.Now().Unix()
+
+	data.WalkMetaNodes(func(node *NodeInfo) {
+		row.Values = append(row.Values, []interface{}{timestamp, node.Status, node.Host, node.ID, METANODE})
+	})
+	data.WalkDataNodes(func(node *DataNode) {
+		row.Values = append(row.Values, []interface{}{timestamp, node.Status, node.Host, node.ID, DATANODE})
+	})
+
+	return []*models.Row{row}
+}
+
+func (data *Data) ShowClusterStatusWithCondition(nodeType, nodeID string, ID uint64) (models.Rows, error) {
```

Comment:
`nodeID` and `ID` is the same thing?

```suggestion
func (data *Data) ShowClusterStatusWithCondition(nodeType string, ID uint64) (models.Rows, error) {
```